python-focal/3.10/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM socrata/base-focal as python_build
+FROM ubuntu:20.04 as python_build
```

Comment:
this makes me nervous. @dosieckityler said in teams recently:

> We pretty much tell people never to run non-socrata images on any of our infra for fear it could interfere with stuff or be vulnerable in weird ways.  

([source](https://teams.microsoft.com/l/message/19:756b9311a756484986ba0184b6502317@thread.tacv2/1763152648408?tenantId=7cc5f0f9-ee5b-4106-a62d-1b9f7be46118&groupId=d5f4a841-6704-441b-87b2-1ab83b662866&parentMessageId=1763076426705&teamName=D%26I%20-%20TIRE&channelName=tire-fire&createdTime=1763152648408&ngc=true))

isn't the whole point of having our own base images that we control what gets shipped to prod?

Reply:
Yep! So the way this works, is it pulls the ubuntu:20.04 image. Does the python build on that layer. Then later copies the build artifacts onto a socrata/base-focal or socrata/runit-focal image to make a python image.

Reply:
To be clear, the end result is that this produces an image built on base-focal or runit-focal. This is also basically how we make our base and runit images. We pull the official one and then apply customizations to those

Reply:
ah, okay, maybe I'm being overly paranoid and foolish. We base our images off ubuntu anyway:

* [base-bionic](https://github.com/socrata/shipyard/blob/be6452af7c0e54bcd751d7b7c21ad6410085c4a6/base-bionic/Dockerfile#L1)
* [base-focal-ecs](https://github.com/socrata/shipyard/blob/be6452af7c0e54bcd751d7b7c21ad6410085c4a6/base-focal-ecs/Dockerfile#L1)
* [base-focal](https://github.com/socrata/shipyard/blob/be6452af7c0e54bcd751d7b7c21ad6410085c4a6/base-focal/Dockerfile#L1)

Reply:
oh yah, yr comment beat me to it, but I arrived at that same conclusion. Okay, this is good. I approve. Thanks for fixing this Jesse!